src/runners/test_alert_processor.py

```diff
@@ -0,0 +1,38 @@
+from runners import alert_processor
+from runners.helpers import db
+import os
+import pytest
+
+CTX = db.connect()
+
+
+def preprocess():
+    alert_processor.main()
+
+
+def processor_test_1():
+    query = "select * from results.alerts where alert:ACTOR = 'test_actor' and suppressed = false"
+    rows = list(db.fetch(CTX, query))
+
+    assert len(rows) == 2
+
+    a1 = rows[0]
+    a2 = rows[1]
+
+    assert len(a1['CORRELATION_ID']) > 5
+    assert len(a2['CORRELATION_ID']) > 5
+    assert a1['CORRELATION_ID'] == a2['CORRELATION_ID']
+
+
+@pytest.mark.run(order=3)
+def test():
+    try:
+        if os.environ['TEST_ENV'] == 'True':
+            preprocess()
+            processor_test_1()
+    except Exception:
+        assert 1 == 0
```

Comment:
why re-raise as an assert? I think the test suite should be relatively smart about exceptions that are raised